ot/bregman.py

```diff
@@ -1214,15 +1404,15 @@ def unmix(a, D, M, M0, h0, reg, reg0, alpha, numItermax=1000,
 
     Parameters
     ----------
-    a : ndarray, shape (d)
+    a : ndarray, shape (n_observed)
```

Comment:
dim_a (its an histogram)

Reply:
Yes this docstring was not accurate. I checked the paper to set the right names, tell me if it's okay now